index.html

```diff
@@ -0,0 +1,180 @@
+      <hr>
+      <label for="username"><b>Username</b></label>
+      <input type="text" placeholder="Enter Username" name="username" required>
+
+      <label for="email"><b>Email</b></label>
+      <input type="text" placeholder="Enter Email" name="email" required>
+
+      <label for="psw"><b>Password</b></label>
+      <input type="password" placeholder="Enter Password" name="psw" required>
+
+      <label for="psw-repeat"><b>Repeat Password</b></label>
+      <input type="password" placeholder="Repeat Password" name="psw-repeat" required>
+      <div class="clearfix">
+        <button type="button" onclick="document.getElementById('id02').style.display='none'" class="cancelbtn">Cancel</button>
+        <button type="submit" class="signupbtn">Sign Up</button>
+      </div>
+    </div>
+  </form>
+</div>
+      </div>
+    </section>
+
+    <section id="newsletter">
+      <div class="container"></div>
+    </section>
+
+      <div class="container">
+      
+          <table border="0" width="90%">
+              <th> <h1>Top Questions</h1></th>
+             <td> 
+               
+              <h3> <button onclick="document.getElementById('id03').style.display='block'" style="width:auto;">Ask Question</button></h3>
+              <div id="id03" class="modal">
+  
+                  <form class="modal-content animate" action="/action_page.php">
+                    <div class="imgcontainer">
+                      <span onclick="document.getElementById('id03').style.display='none'" class="close" title="Close modalee">&times;</span>
+                    </div>
+                
+                    <div class="container">
+                      <label for="uname" align="center"><b>Your Question</b></label> <br/>
+                      <textarea rows = "5" cols = "115" name = "description">
+                        
+                       </textarea>
+                        
+                      <button type="submit">Ask</button>
+                      
+                    </div>
+                
+                    <div class="container" style="background-color:cadetblue">
+                      <button type="button" onclick="document.getElementById('id03').style.display='none'" class="cancelbtn">Cancel</button>
+                      
+                    </div>
+                  </form>
+                </div>
+            </td>
+             </table>
+
+             <table>
+               
+                <tr>
+                    <div class="votes">
+                        <a href="#">vote</a>
+                    </div>
+    
+                    <div class="votes">
+                     <a href="#">answer</a> 
+                    </div>
+    
+                    <div class="votes">
+                        <a href="#">views</a>
+                    </div>
+             </table>
+
+      </div>
+
+    <footer>
+      <p>StackOverFlow-Lite Copyright &copy; 2019</p>
+    </footer>
+    <script>
```

Comment:
Don't use inline styles. They cannot be cached by the browser and this might impact performance.

Extract it to a separate file and then load it using a script with an src attribute

Reply:
correction made